github/users.go

```diff
@@ -134,6 +134,55 @@ func (s *UsersService) Edit(ctx context.Context, user *User) (*User, *Response,
 	return uResp, resp, nil
 }
 
+// UserContextualInfoOptions specifies optional parameters to the UsersService.GetContextualInfo
+// method.
+type UserContextualInfoOptions struct {
+	// SubjectType specifies the additional information to be received about the hovercard.
+	// Possible values are : organization, repository, issue, pull_request.
+	SubjectType string `url:"subject_type,omitempty"`
+
+	// SubjectID specifies the ID for the SubjectType.
+	SubjectID string `url:"subject_id,omitempty"`
```

Comment:
Both the parameters are required, not optional. So remove `,omitempty` and add `(Required.)` at the end of comment (like [here](https://gotools.org/github.com/google/go-github/github?rev=88eb4e9dc91cb2db1bbc56c936f1ee39bbe72ba5#orgs-teams.go-L118)).